src/jobs/sonarqube-audit.yml

```diff
@@ -0,0 +1,24 @@
+description: >
+  Job runs SonarQube scanner and upload results
+
+executor: node
+
+parameters:
+  sonarqube_settings_path:
+    type: string
+    description: "Path to file with sonarqube settings"
+
+steps:
+  - checkout
+  - attach_workspace:
+      at: .
+  - run:
+      name: Install Sonarqube scanner
+      command: |
+        wget https://binaries.sonarsource.com/Distribution/sonar-scanner-cli/sonar-scanner-cli-4.5.0.2216-linux.zip
+        unzip sonar-scanner-cli-4.5.0.2216-linux.zip
+  - run:
+      # it will ignore if there is an error runnign the scanner
+      name: Run Sonarqube scanner
+      command: |
+        ./sonar-scanner-4.5.0.2216-linux/bin/sonar-scanner -Dproject.settings=<< parameters.sonarqube_settings_path >> || true
```

Comment:
Could we remove the non-blocker step `... || true`, and leave that to users of the orb, so that they can decide we should be a blocker or not

Reply:
Done